soc/infineon/cat1b/cyw20829/bootstrap.ld

```diff
@@ -0,0 +1,171 @@
+/* Copyright 2024 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/devicetree.h>
+#include <zephyr/linker/sections.h>
+
+/* ------------------------------------------------------------------------- */
+
+/* Memory reserved for Bootstrap code and data */
+BOOTSTRAP_SIZE = DEFINED(APP_BOOTSTRAP_SIZE) ? APP_BOOTSTRAP_SIZE : 0x00002400 ;
+
+FLASH_START_ADDR_SAHB       = 0x60000000;
+FLASH_START_ADDR_CBUS       = 0x08000000;
+FLASH_END_ADDR_SAHB         = 0x60080000; /* 512K */
+
+RAM_START_ADDR_SAHB         = 0x20000000;
+RAM_END_ADDR_SAHB           = 0x20040000; /* 256K */
+RAM_START_ADDR_CBUS         = 0x20000000;
+
+BOOTSTRAP_OFFSET_FLASH      = 0x00000050; /* toc2=0x10, l1_desc=0x1C, sign_header=0x20 */
+
+FLASH_ALIGNMENT_SIZE        = 0x00000200;
+
+/* vma for bootstrap code region */
+CODE_VMA                    = RAM_START_ADDR_CBUS + (RAM_END_ADDR_SAHB - RAM_START_ADDR_SAHB - BOOTSTRAP_SIZE); /* 0x2003DC00 */
+/* lma for bootstrap code region */
+CODE_LMA                    = FLASH_START_ADDR_SAHB + BOOTSTRAP_OFFSET_FLASH; /* 0x60000050 */
+/* Maximum bootstrap code + data size */
+CODE_BS_SIZE                = BOOTSTRAP_SIZE; /* 9KB */
+/* vma for bootstrap data region */
+DATA_BS_VMA                 = RAM_END_ADDR_SAHB - BOOTSTRAP_SIZE; /* 0x2003DC00 */
+/* vma for bootstrap and app data region */
+DATA_VMA                    = RAM_START_ADDR_SAHB; /* 0x20000000 */
+/* vma for appCodeRam region */
+DATA_CBUS_VMA               = RAM_START_ADDR_CBUS; /* 0x20000000 */
+/* lma for bootstrap and app data region */
+DATA_LMA                    = CODE_LMA + CODE_BS_SIZE; /* 0x60002450 */
+/* data size */
+DATA_SIZE                   = RAM_END_ADDR_SAHB - DATA_VMA - BOOTSTRAP_SIZE; /* 0x3DC00 */
+
+/* Size of Bootstrap data is kept same as BOOTSTRAP_SIZE */
+DATA_BS_SIZE                = BOOTSTRAP_SIZE;
+
+/* ------------------------------------------------------------------------- */
+/* Boot strap section */
+
+MEMORY
+{
+    BS_CODE (rwx) : ORIGIN = CODE_VMA, LENGTH = CODE_BS_SIZE
+    BS_DATA (rwx) : ORIGIN = DATA_BS_VMA,  LENGTH = DATA_BS_SIZE
```

Comment:
With the hardcoded configuration above, these regions seems to be identical (=sram_app_header in DT).